packages/pancake-uikit/src/components/Svg/Icons/CircleOutline.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
```

Comment:
Being a sneaky bish and trying to sneak an unrelated (Competition page) svg into this PR 🙏